ckanext/canada/tables/briefingt.yaml

```diff
@@ -113,8 +113,11 @@ resources:
         en: Minister
         fr: Ministre
       D:
-        en: deputy head (including a person appointed to a position of an equivalent rank)
-        fr: Administrateurs Généraux (y compris une personne nommée à un poste de rang équivalent)
+        en: Deputy head (including a person appointed to a position of an equivalent rank)
+        fr: Administrateurs généraux (y compris une personne nommée à un poste de rang équivalent)
+        d_short:
```

Comment:
how abour `shortform:` instead? The `d` is implied by it being part of the `D:` value above

Reply:
```suggestion
        shortform:
```

Reply:
also I thought the lowercase `deputy head` text was specifically requested (part of change ffbe2b691c )